shared/src/bundles/user-details/enums/user-details-api-path.enum.ts

```diff
@@ -1,7 +1,8 @@
 const UserDetailsApiPath = {
     ROOT: '',
     $ID: '/:userId',
-    APPROVE: '/approve',
+    APPROVE: '/approve/:userId',
+    DENY: '/deny/:userId',
```

Comment:
Change to /:userId/approve, because first we specify to which user we need to do action and after that action. It is more clear